docs/articles/sync-to-main-repo.md

```diff
@@ -0,0 +1,30 @@
+<p>Here, I am assuming that you have already connected your GitHub account with the Visual Studio Code.</p>
```

Comment:
Why html?

It should be markdown

Reply:
Converted to markdown @proful 
It is showing this
![Pic1](https://user-images.githubusercontent.com/72906055/96961306-f268f180-1521-11eb-9d89-4823fe280138.JPG)